### PR TITLE
Core 831 update selenium axe driver

### PR DESCRIPTION
**Pull request by nejoyer**
See commits

This is a csharp wrapper, specifically for selenium, around the axe javascript tool for ada analysis on web pages.
There is a java wrapper here (https://github.com/dequelabs/axe-selenium-java/blob/master/src/main/java/com/deque/axe/AXE.java) that is provided by the company that supports Axe. However, we use csharp. I found https://github.com/javnov/axe-selenium-csharp which was a good start on a csharp one, but wanted to improve it.

I've forked it into the AF org so that we can update without relying on the original creator approving PRs, but I'll probably issue a PR from this fork into that repo if we like what I have here.

### CODE REVIEW

File: Globant.Selenium.Axe/Globant.Selenium.Axe.Test/IncludeExcludeManagerTests.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+            // act
+            var contextJson = includeExcludeManager.GetContextJson();
+
+            //Assert
+            contextJson.Should().BeEquivalentTo("{\"include\":[[\"foo\"]],\"exclude\":[[\"bar\"]]}");
+        }
+
+        [TestMethod]
+        public void GivenThatOneSelectorIsIncludedFromAnIFram_WhenIGetContextJson_ThenTheResultShouldHaveTheSelectorAndIFrameSelectorInTheRightFormat()
```
Comment (BenWaller) [inline Globant.Selenium.Axe/Globant.Selenium.Axe.Test/IncludeExcludeManagerTests.cs]: IFram -> IFrame

**Review by BenWaller — APPROVED** [review]

File: Globant.Selenium.Axe/Globant.Selenium.Axe.Test/OptionsManagerTests.cs [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            // act
+            var optionsJson = OptionsManager.GenerateOptionsJson();
+
+            //Assert
+            optionsJson.Should().BeEquivalentTo("{\"rules\":{\"a\":{\"enabled\":true},\"b\":{\"enabled\":true},\"c\":{\"enabled\":false},\"d\":{\"enabled\":false}},\"runOnly\":{\"type\":\"tag\",\"values\":[\"tag1\"]}}");
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(InvalidOperationException))]
+        public void GivenIncludedAndExcludedRulesAndNoTags_WhenIGenerateOptionsJson_ThenTheCorrectJsonIsReturned()
```
Comment (vue-tou) [inline Globant.Selenium.Axe/Globant.Selenium.Axe.Test/OptionsManagerTests.cs]: ...ThenAnInvalidOperationtionExceptionOccurs

**Review by vue-tou — COMMENTED** [review]
Just one minor naming comment.

It'll be interesting to see this in action for online ADA testing.

**Comment by nejoyer** [comment]
You guys have some eagle eyes!

**Comment by nejoyer** [comment]
Whoops. Didn't mean to close... just meant to comment.

**Pull request merged**